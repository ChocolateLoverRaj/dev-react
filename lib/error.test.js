/* global test, expect */

import ErrorWrapper from './error.js'

test('constructing', () => {
  const error = new Error('this will be wrapped')
  const errorWrapper = new ErrorWrapper('something didn\'t work', error)
  expect(errorWrapper.originalError).toBe(error)
  expect(errorWrapper.message).toMatchSnapshot()
})
