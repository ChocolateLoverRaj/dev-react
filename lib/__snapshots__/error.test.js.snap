// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`constructing 1`] = `
"something didn't work

Original: Error: this will be wrapped
    at Object.<anonymous> (C:\\\\Users\\\\Jui\\\\Documents\\\\GitHub\\\\dev-react\\\\lib\\\\error.test.js:6:17)
    at Object.asyncJestTest (C:\\\\Users\\\\Jui\\\\Documents\\\\GitHub\\\\dev-react\\\\node_modules\\\\jest-jasmine2\\\\build\\\\jasmineAsyncInstall.js:106:37)
    at C:\\\\Users\\\\Jui\\\\Documents\\\\GitHub\\\\dev-react\\\\node_modules\\\\jest-jasmine2\\\\build\\\\queueRunner.js:45:12
    at new Promise (<anonymous>)
    at mapper (C:\\\\Users\\\\Jui\\\\Documents\\\\GitHub\\\\dev-react\\\\node_modules\\\\jest-jasmine2\\\\build\\\\queueRunner.js:28:19)
    at C:\\\\Users\\\\Jui\\\\Documents\\\\GitHub\\\\dev-react\\\\node_modules\\\\jest-jasmine2\\\\build\\\\queueRunner.js:75:41
    at processTicksAndRejections (internal/process/task_queues.js:93:5)"
`;
